data/remnant/remnant news.txt

```diff
@@ -0,0 +1,279 @@
+	word
+		"00"
+		"01"
+		"03"
+		"04"
+		"05"
+		"06"
+		"07"
+
+
+
+phrase "00-07 days"
+	phrase
+		"00-07"
+	word
+		" days"
+
+
+
+phrase "0-F"
+	word
+		"0"
+		"1"
+		"2"
+		"3"
+		"4"
+		"5"
+		"6"
+		"7"
+		"8"
+		"9"
+		"A"
+		"B"
+		"C"
+		"D"
+		"E"
+		"F"
+
+
+
+phrase "00-1F"
+	word
+		"0"
+		"1"
+	phrase
+		"0-F"
+
+
+
+phrase "A group of"
+	word
+		"A "
+	word
+		"pair "
+		"couple "
+		"pack "
+		"squad "
+		"team "
+		"family "
+		"group "
+		"gathering "
+		"small group "
+		"medium group "
+		"large group "
+	word
+		"of"
+
+
+
+phrase "Indeterminate"
+	word
+		"A few "
+		"Several "
+		"Some "
+		"Lots of "
+		"Many "
+
+
+
+phrase "Amount of Remnant"
+	phrase
+		"A group of"
+		"Indeterminate"
+	word
+		"Remnant"
+
+
+news "remnant security advisory"
+	location
+		government "Remnant"
+	name
+		word
+			"System Security "
+			"Local "
+			"System "
+			"Defense "
+		word
+			"Advisory"
+			"Status"
+			"Threats"
+	message
+		word
+			"Incoming Korath Fleet in "
+		phrase
+			"00-07 days"
+		word
+			" "
+		phrase
+			"00-1F"
+		word
+			" hours. Threat Sierra-"
+		word
+			"2"
+			"3"
+		phrase
+			"0-F"
+		word
+			"."
+	message
+		word
+			"Incoming Korath Ship in "
+		phrase
+			"00-07 days"
+		word
+			" "
+		phrase
+			"00-1F"
+		word
+			" hours. Threat Sierra-"
+		word
+			"4"
+			"5"
+		phrase
+			"0-F"
+		word
+			"."
+	message
+		word
+			"Patrol "
+			"Salvage "
+			"Sentry "
+			"Attack "
+			"Reconnaissance "
+			"Escort "
+			"Decoy "
+		word
+			"Flight "
+			"Talon "
+			"Wing "
+		phrase
+			"0-F"
+		phrase
+			"0-F"
+		phrase
+			"0-F"
+		word
+			" prepare for "
+			" ready for "
+			" standby for "
+			" hold for "
+		word
+			"departure."
+			"launch."
+			"combat."
+	message
+		word
+			"Salvage ships report to orbit for incoming Korath ships."
+			"No change in current threat level."
+			"Maintain normal security status."
+			"All departing ships advised to have weapon systems hot."
+			"All departing ships advised to expect hostiles in the vicinity."
+			"Reminder to update your IFF settings and verify keys."
+			"Reminder to verify functionality of selective IFF broadcast every startup."
+
+
+
+news "remnant incomprehensible"
+	location
+		government "Remnant"
+	name
+		word
+			"Spaceport"
+			"Loading dock"
+			"Cafeteria"
+			"Landing pad"
+			"Landing dock"
+			"Spaceport alley"
+			"Spaceport hallway"
+		word
+			" scene"
+	message
+		phrase 
+			"Amount of Remnant"
+		word
+			" sign "
+			" gesture "
+		word
+			"quickly "
+			"calmly "
+			"leisurely "
+			"slowly "
+			"deliberately "
+			"carefully "
+			"precisely "
+			"sharply "
+			"bruskly "
+			"briskly "
+			"sweepingly "
+			"intensively "
+			"meticulously "
+			"at length "
+			"vaguely "
+			"briskly "
+			"hurriedly "
+			"hastily "
+			"abruptly "
+			"spontaneously "
+		word
+			"to "
+		word
+			"each other. "
+			"another Remnant. "
+			"other Remnant. "
+			"a Remnant at the information desk. "
+			"a dockworker."
```

Comment:
```suggestion
			"a dockworker. "
```

Reply:
Better to just have the `. ` be a different word than to forget it in one of these words.